module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -1917,9 +1921,12 @@ public function resetPassword(array $details, array $params)
             ]
         );
         if ($result['code'] >= 300) {
+            $msg = empty($result['data']['error'])
+                ? $this->getPrefixedMessage($result['code'])
+                : $this->getPrefixedMessage($result['data']['error']);
             return [
                 'success' => false,
-                'error' => $result['data']['error'] ?? $result['code'],
+                'error' => $msg,
```

Comment:
If we're going to prefix both the code and the error message, we don't really need the more complex `$msg` assignment above, and we can just do this:

```suggestion
                'error' => $this->getPrefixedMessage($result['data']['error'] ?? $result['code']),
```